mappings/net/minecraft/util/Nameable.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/class_6439 net/minecraft/util/Nameable
 	METHOD method_25830 getDisplayName ()Lnet/minecraft/class_6244;
+	METHOD method_29611 getCustomName ()Ljava/lang/String;
```

Comment:
getName

Reply:
`hasCustomName` exists

Reply:
Should be `getName`, like in `CommandSource`. `hasCustomName` just says whether the name from `getName` is custom, see `Entity::getName` which has
```java
if (this.hasCustomName()) {
    return this.getCustomName();
} else {
    ...
}
```